packages/native-core-macro/src/lib.rs

```diff
@@ -22,12 +23,107 @@ pub fn sorted_str_slice(input: TokenStream) -> TokenStream {
 }
 
 #[derive(PartialEq, Debug, Clone)]
-enum DepKind {
+enum DependancyKind {
```

Comment:
typically spelled "dependency"

"dependancy" looks real weird to me, but is apparently a valid alternative spelling...

Reply:
Oops, good catch